docs/docs/tutorial/index.md

```diff
@@ -263,6 +264,26 @@ echo $(cat credentials/bob-funds.addr)"\n"
 
 In case you don't have test ada on `preprod`, you can use the [testnet faucet](https://docs.cardano.org/cardano-testnets/tools/faucet/) to fund your wallet or the addresses above. Note that due to rate limiting, it's better to request large sums for efficiency and distribute as needed.
 
+Something like the following (if you used the faucet to give funds only to `alice-funds.addr`):
+
+```
+# Build a Tx to send funds from `alice-funds` to the others who need them: bob
+# funds and nodes.
+cardano-cli transaction build \
+    --tx-in <the TxId of the faucet Tx that provided the funds> \
```

Comment:
Maybe I should include the jq command here to extract this address from the network?